test/webhook-proxy.test.ts

```diff
@@ -50,12 +56,13 @@ describe('webhook-proxy', () => {
     const url = 'http://bad.proxy/events'
     nock('http://bad.proxy').get('/events').reply(404)
 
-    const log = logger.child()
+    const options = {}
+    const log = logger.child(options)
```

Comment:
`logger.child()` was complaining that it didn't receive its [required argument](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/3ee3f99b0011b6541ef959868b970b256bca5460/types/bunyan/index.d.ts#L15). Adding `const options = {}` and passing `options` into the function was the only way to appease TypeScript here. 🤔 (tried just calling `logger.child({})` but that wasn't working)

Reply:
I think this could also be:
```ts
const log = logger.child(options: {})
```